frontend/src/components/pages/topics/Tab.Messages.tsx

```diff
@@ -405,14 +424,19 @@ export class TopicMessageView extends Component<{ topic: TopicDetail }> {
                 onFilterDropdownVisibleChange: (_) => this.showColumnSettings = true,
                 filterIcon: (_) => {
                     return <Tooltip title='Column Settings' mouseEnterDelay={0.1}>
-                        <SettingFilled />
+                        <SettingFilled style={IsColumnSettingsEnabled ? { color: '#1890ff' } : { color : '#a092a0' }}/>
                     </Tooltip>
                 },
                 render: (text, record) => !record.isValueNull && (
                     <span>
                         <ZeroSizeWrapper width={32} height={0}>
```

Comment:
1. Icon is not vertically centered (but should be):
   ![](https://i.imgur.com/FA8q8T7.png)


2. Clickable region of the button should fill the whole table-cell; remove padding in those (msgTableActionColumn) cells

    green: currently clickable
    red: should be clickable
    ![](https://i.imgur.com/8dyWZnJ.png)

Reply:
I'd probably do something like this:
- remove usage of `<ZeroSizeWrapper>` here
- `.ant-table.ant-table-middle .ant-table-tbody>tr>td.msgTableActionColumn { padding: 0 }`
- ensure button fills the whole parent
- if the alignment of the icon inside the button is an issue, you can try making the button a flex container: something like `<Button style={{display:'flex', placeContent:'center'}} ...` or so